Scripts/jq-canonify-nodestate

```diff
@@ -0,0 +1,11 @@
+#! /usr/bin/env bash
+#
+# A zeek-client output filter for get-nodes and related commands that filters
+# stdin to stdout as follows:
+#
+# - removes .pid entries, since they will differ in every run
+#
+# - normalizes cluster node states PENDING to RUNNING, for tests that do not
+#   care which of those two states a node is in.
+
+jq 'del(.results[][].pid) | .results[][] |= if (.state == "PENDING") then (.state = "RUNNING") else . end'
```

Comment:
Could we canonify RUNNING and PENDING to something else but a valid state? Maybe "<removed>" or "<stripped>"?

Mostly worried about the poor person looking at the output and making conclusions/assumptions on the difference between pending and running, the above seems harmful. A `<stripped>` will immediately indicate the actual state was fiddled with.

Reply:
I thought about that too and didn't feel strongly about it, mainly because you need to understand the tests anyway in order to work with them. I'm fine with changing it.